Sources/Automerge/scalar.swift

```diff
@@ -2,16 +2,27 @@ import enum AutomergeUniffi.ScalarValue
 
 typealias FFIScalar = AutomergeUniffi.ScalarValue
 
+/// A type that represents a primitive Automerge value
 public enum ScalarValue: Equatable, Hashable {
+    /// A byte buffer
     case Bytes([UInt8])
+    /// A string
     case String(String)
+    /// An unsigned integer
     case Uint(UInt64)
+    /// A signed integer
     case Int(Int64)
+    /// A floating point number
     case F64(Double)
+    /// An integer counter
     case Counter(Int64)
+    /// A timestamp
     case Timestamp(Int64)
+    /// A Boolean value
     case Boolean(Bool)
+    /// An unknown, raw scalar type.
```

Comment:
This type basically exists for forwards compatibility. Might be worth mentioning that it's not expected that users would construct this case

Reply:
I'll make an explicit note of it being a reserved type.